crates/net/src/driver.rs

```diff
@@ -32,6 +32,16 @@ impl NetworkDriver {
         NetworkDriverBuilder::new()
     }
 
+    /// Take the unsafe block receiver.
+    pub fn take_unsafe_block_recv(&mut self) -> Option<Receiver<ExecutionPayloadEnvelope>> {
+        self.unsafe_block_recv.take()
+    }
+
+    /// Take the unsafe block signer sender.
+    pub fn take_unsafe_block_signer_sender(&mut self) -> Option<watch::Sender<Address>> {
```

Comment:
I don't fully understand what this is for. Is it so that eventually we can subscribe to logs on the L1 to read updates to the p2p block signer on the fly?

Reply:
Yes exactly - I'm not sure if we ever need it, but good to provide a way to access for now imo